torch/csrc/utils/disable_torch_function.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <torch/csrc/python_headers.h>
+
+namespace torch {
+  bool torch_function_enabled();
```

Comment:
A comment here or in the cpp file on why there's a disabling mechanism and where to turn it on/off would be useful. And is this exposed to users or internal?